api/api/utils/relationship.py

```diff
@@ -9,3 +9,13 @@ def get_relationship(relationship: str) -> Relationship:
     if relationship == "default":
         return Relationship.DEFAULT
     raise ValueError(f"Relationship {relationship} is not a valid relationship")
+
+
+def get_relationship_str(relationship: Relationship) -> str:
```

Comment:
idk if we need this? Why not just `str(relationship)` instead of a util function for it?

Reply:
oh because its a float, hmm